packages/ui/tailwind-utils-config/components/textarea.ts

```diff
@@ -0,0 +1,108 @@
+import { CSSRuleObject } from 'tailwindcss/types/config'
+
+// themes can be extended in the future
+const themes = ['danger'] as const
+
+function createInputThemeStyles() {
+  const styles: CSSRuleObject = {}
+
+  themes.forEach(theme => {
+    styles[`&:where(.cn-textarea-${theme}):not(textarea[disabled])`] = {
```

Comment:
Little enhancement: 

```suggestion
    styles[`&:where(.cn-textarea-${theme}):not(:disabled)`] = {
```

Reply:
In `input` we had like `:has(input[disabled])`, because we were applying styles for the container. As we are applying styles to `textarea` directly, we can use `:disabled`.

Reply:
Fixed